convert2rhel/pkgmanager.py

```diff
@@ -14,10 +14,16 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import sys
+
 
 try:
     from yum import *
 
+    # Yum's installation path for the CLI code
+    # https://github.com/rpm-software-management/yum/blob/4ed25525ee4781907bd204018c27f44948ed83fe/bin/yum#L26
+    sys.path.insert(0, "/usr/share/yum-cli")
```

Comment:
This will be imported every time `yum` is detected, should I move it to here? https://github.com/oamg/convert2rhel/blob/11bab92119510bb40eb7e9c9bc9d0010f7122ff7/convert2rhel/pkghandler.py#L916 For now, this line is only used by this function.

Reply:
As it is now the `sys.path.insert(0, "/usr/share/yum-cli")` will be executed just once because the pkgmanager.py is loaded just once per convert2rhel execution. But moving it to the pkghandler.py makes sense from the point of view that it logically relates just to the one `_get_packages_to_update_yum()` function.